src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -435,26 +435,29 @@ public Nsi getNsiByNsiId(
         return offenderService.getAllPrimaryIdentifiers(filter, pageable);
     }
 
-    @ApiIgnore("This endpoint is is to be removed as soon as court case service has moved to the /offenders/crn/{crn}/convictions/{convictionId}/sentences/{sentenceId}/status version")
-    @Deprecated(forRemoval = true)
-    @ApiOperation(value = "Return sentence and custodial status information by crn, convictionId and sentenceId.")
+    @ApiOperation(value = "Return sentence and custodial status information by crn, convictionId.")
     @ApiResponses(
         value = {
             @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
-            @ApiResponse(code = 404, message = "The offender CRN / conviction ID / sentence ID is not found", response = ErrorResponse.class),
+            @ApiResponse(code = 404, message = "The offender CRN / conviction ID is not found", response = ErrorResponse.class),
             @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
         })
-    @GetMapping(path = "/offenders/crn/{crn}/convictions/{convictionId}/sentences/{sentenceId}/custodialStatus")
-    public SentenceStatus getSentenceStatusBySentenceIdToRemove(
+    @GetMapping(path = "/offenders/crn/{crn}/convictions/{convictionId}/sentenceStatus")
```

Comment:
so this replaces the deprecated one you no longer use?

Reply:
It will - the other is still there while we migrate off